web/wp-content/themes/dctx-v2/page.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-oytbbs.php, but found phpcs-scan-oytBbs.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).